_templates/how-to.md

```diff
@@ -0,0 +1,47 @@
+---
+title: [Title]
+description: [Brief description of the How-To article]
+type: [HowTo]
+<!-- TO DO: Do we we need the below fields for How-To articles? -->
+[teampulseid: [number]
+[ticketid: [number]
+[pitsid: [number] <!-- Not sure we need this, as pits item are related to teampulse anyway -->
+<!-- TO DO: What else should we have here? -->
+---
+
+# Title
```

Comment:
We drop the whole #Title filed as it will be automatically populated from the meta title tag. We will add the description there. @ikoevska check the layout!